src/model/model.rs

```diff
@@ -98,6 +98,58 @@ pub struct Command {
     pub payload: Vec<u8>,
 }
 
+//TODO[@humb1t:RH2-16]: rename
+pub enum Relation {
+    /// Belongs to account with defined identification.
+    /// For example we can fill a map of accounts to assets by this relation.
+    BelongsTo(String),
+    GoingTo(String),
+    GoingFrom(String),
+}
+
+/// This trait should be implemented for commands with `account_id` field.
+/// Marking your command with `impl` of this trait you provide an ability
+/// to retrieve information about relation to an account.
+//TODO[@humb1t:RH2-16]: name is very bad, should be renamed.
+pub trait Accountability {
+    fn relations(&self) -> Vec<Relation>;
+}
+
+impl Accountability for Command {
+    //TODO: implement
+    fn relations(&self) -> Vec<Relation> {
+        use Relation::*;
+        match &self.command_type {
+            17 => {
+                let command: commands::TransferAsset = self.payload.clone().into();
+                vec![
+                    GoingFrom(command.source_account_id.clone()),
+                    GoingTo(command.destination_account_id.clone()),
+                    BelongsTo(command.destination_account_id.clone()),
+                ]
+            }
+            _ => Vec::new(),
+        }
+    }
+}
+
+pub trait Assetibility {
```

Comment:
We should probably also think of a new name for this :)

Reply:
Yeah, I made an additional issue for this)